Python/pywarpx/picmi.py

```diff
@@ -416,6 +416,8 @@ def initialize_inputs(self, species_number, layout, species, density_scale):
         species.flux_normal_axis = self.flux_normal_axis
         species.surface_flux_pos = self.surface_flux_position
         species.flux_direction = self.flux_direction
+        species.flux_tmin = self.flux_tmin
```

Comment:
This is relying on `flux_tmin` and `max` being added as attributes to `picmistandard.PICMI_UniformFluxDistribution`. Is there going to be a PR in picmi to add these?

Reply:
Did you mean this PR?
https://github.com/picmi-standard/picmi/pull/98/files
Or do we need something in addition to this?

Reply:
Sorry, yes, never mind. My local repo wasn't updating properly and I didn't see the options (now fixed).